Doc: trip delete (#3081)
CanalTP/navitia: documentation/slate/source/includes/realtime.md

Real time isn't available on every coverage for Navitia. For real time to be available for a client, it needs to provide real time info about its network to Navitia.
</aside>

> The effect of a disruption can be among of the following:

Reviewer: "One of the following" or "among the following", not "among **of** the following".

> The effect of a disruption can be among of the following:

Reviewer 2: i think it's 'among the following'

Author: Already corrected, @mbounabi called dips on that one :stuck_out_tongue:

<ul>
    <li>[SIGNIFICANT_DELAYS](#SIGNIFICANT_DELAYS)</li>
    <li>[REDUCED_SERVICE](#REDUCED_SERVICE)</li>
    <li>[NO_SERVICE](#NO_SERVICE)</li>
    <li>[MODIFIED_SERVICE](#MODIFIED_SERVICE)</li>
    <li>[ADDITIONAL_SERVICE](#ADDITIONAL_SERVICE)</li>
    <li>[UNKNOWN_EFFECT](#UNKNOWN_EFFECT)</li>
</ul>

> This list follows the values of the GTFS RT documentation at <https://gtfs.org/reference/realtime/v2/#enum-effect>.

Reviewer 3: ```suggestion
This list follows the GTFS RT values documented at <https://gtfs.org/reference/realtime/v2/#enum-effect>.
```

> List follows the values of the GTFS RT documentation at <https://developers.google.com/transit/gtfs-realtime/reference#Effect>.

Reviewer: ```suggestion
This list follows the values of the GTFS RT documentation at <https://developers.google.com/transit/gtfs-realtime/reference#enum-effect>.
```

If "data_freshness" is "base_schedule", then the depature time is displayed.  
In that case, a link to this disruption can be found in the section "display_informations" for departures, in the "date_times" object itself for stop_schedules.

## <a name="NO_SERVICE"></a>No service

![image](no_service.png)

``` shell
# Extract of an impacted trip from /disruptions
{
    "application_periods": ⊖[
       ⊖{
            "begin": "20191210T183100",
            "end": "20191210T194459"
        }
    ],
    "impacted_objects": ⊖[
       ⊖{
            "pt_object": ⊖{
                "embedded_type": "trip",
                "id": "OCE:BOA017534OCESNF-20191210",
                "name": "OCE:BOA017534OCESNF-20191210",
                "trip": ⊕{2 items}
            }
        }
    ],
    "severity": ⊖{
        "effect": "NO_SERVICE"
        "name": "trip canceled",
    },
}
```
The effect of the disruption is `NO_SERVICE`. It means that the train won't be circulating at all.

> In the disruption, the deleted trip can be found in the list of "impacted_objects" with the

Reviewer 3: ```suggestion
In the disruption, the deleted trip can be found in the "impacted_objects" list with the
```

"application_periods" describing the period(s) of unavailability for the trip.  
See the [disruption](#disruption) objects section for its full content and description.

<div></div>
### Journeys

> If the trip deleted is used in part by a section of the `base_schedule` journey, Navitia will compute a different

Reviewer 3: ```suggestion
If the deleted trip is used by a section of the `base_schedule` journey, Navitia will compute a different
```

> itinerary to the requested station in `realtime` (without using that trip).  

Reviewer 3: I'm not sure this makes sense to me. Do you have an example to illustrate the case ?

Author: Ex: You plan a **base-schedule** journey from La Défense to Chatelet in Paris.
That journey uses a trip (VJ) from line A.
If that very trip is deleted (in realtime), when computing a **realtime** journey, you will use a different itinerary: either a later trip from line A, or line 1 if it becomes better.

Hope it clarifies the case. If you think of a better way to explain it, glad to add/change it !
